RELEASES.md

```diff
@@ -0,0 +1,49 @@
+# Releases
+
+This document describes how Strimzi handles versioning, releases new versions and communicates about them.
+
+Strimzi uses [semantic versioning](https://semver.org/) as a versioning schema for the releases of all the components.
+
+## Release Cycle
+
+Strimzi is using a release schedule of 1 or 2 months for the "core" component, the cluster operator.
+The [GitHub milestones](https://github.com/strimzi/strimzi-kafka-operator/milestones) are used to track the features and bug fixes included in each release.
```

Comment:
Do we want to also point to our pathetic roadmap? https://github.com/orgs/strimzi/projects/1

Reply:
I never know how much it's updated ... shame on me but I don't take a look often to it. Interesting you said it's pathetic :-D ... anyway if you want I can mention it.

Reply:
Well, the roadmap sucks ... and to be honest our planning is bad. But I think anyone seeing this will ask for it. So I think we need to do something with it.

Reply:
Would it help the planning and keeping it updated if it were part of the release process to do so?